pyqtgraph/graphicsItems/PlotItem/PlotItem.py

```diff
@@ -551,6 +552,18 @@ def addItem(self, item, *args, **kargs):
         if name is not None and hasattr(self, 'legend') and self.legend is not None:
             self.legend.addItem(item, name=name)            
 
+        if isinstance(item, GridItem):
+            item.setAxes({ax['item'].orientation: ax['item'] for ax in self.axes.values()})
+            item.setCtrl(self.ctrl)
+
+    def addDataItem(self, item, *args):
```

Comment:
I think we already removed this method; probably should yank it out of this PR.

Reply:
Removed with commit e88d5ab